ksqldb-engine/src/main/java/io/confluent/ksql/engine/SandboxedExecutionContext.java

```diff
@@ -172,7 +173,8 @@ public ExecuteResult execute(
       ).execute(ksqlPlan.getPlan(), restoreInProgress);
 
       // Having a streams running in a sandboxed environment is not necessary
-      if (!getKsqlConfig().getBoolean(KsqlConfig.KSQL_SHARED_RUNTIME_ENABLED)) {
+      if (result.getQuery().isPresent()
```

Comment:
@wcarlson5 is this PR done yet/ready for review? Just because I notice that this config is also used a few lines below this

Reply:
Yes that is about runtimes in general and should be the config